vpr/src/place/place.cpp

```diff
@@ -547,7 +547,7 @@ void try_place(const Netlist<>& net_list,
     initial_placement(placer_opts,
                       placer_opts.pad_loc_type,
                       placer_opts.constraints_file.c_str(),
-                      noc_opts.noc);
+                      noc_opts);
```

Comment:
I suspect we could get noc_opts from a context, so that's a possible refactoring (but you can defer it if it's big or you don't agree).

Reply:
noc_opts is a member variable of t_vpr_setup which is instantiated in main(). Should I move it to NocContext?

Reply:
I just took a look and the routing, placement etc. opts are there too, so let's leave it where it is (in t_vpr_setup)